extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -1066,12 +1010,17 @@ function formatValue( $field, $value ) {
 			case "cl_user_id":
 				$oUser = User::newFromId( $value );
 				$oUser->load();
-				return sprintf("<a href=\"%s\">%s</a>", $oUser->getUserPage()->getLocalUrl(), $oUser->getName());
+				return sprintf( '<a href="%s">%s</a>',
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the code after this "return". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1763)